reservationWeekday.js

```diff
@@ -0,0 +1,18 @@
+const { DateTime } = require("luxon");
+let halls = require("./halls");
+let http = require('http');
+let reservedHalls = [];
+
+for (let i = 0; i < halls.length; i++) {
```

Comment:
Kod for petlje treba imati na umu da je length uvijek za jedan veći od posljednjeg indexa jer indexi počinju s 0. Tako da treba koristiti length - 1.
S druge strane bolje je koristiti filter funkciju jer ima manje mjesta za greške i pregledniji je

Reply:
Doduše, ovdje sam ja previdio da je < a ne <= kao komparator, tako da je ovaj kod u redu.
Svejedno, dobro je imati na umu ovo gore navedeno.